### PR TITLE
Added function for ls, as retriever.lib.datasets

### PR DESCRIPTION
**Pull request by ShivamNegi**
@henrykironde @ethanwhite

### CODE REVIEW

**Comment by henrykironde** [comment]
why are you creating a new module? 
datasets.py

**Comment by ShivamNegi** [comment]
The CLI will call the functions from the python interface. I am not sure about its placement so made a module. Where should I put this function?

**Comment by henrykironde** [comment]
So in this case, how would the call from python look.
That's to say; 
If I were to use retriever in one of my projects as a library/package, 
How would the import statements of the dataset() function look like after the final python interface implementation?

**Comment by ShivamNegi** [comment]
As you can see it currently looks like 
```python
>>>import retriever.lib.datasets as retls
>>>retls.datasets()
```
I was thinking of adding all such function imports in the ```__init__.py```. So that later the import will be simpler.
```python
>>>from retriever.lib import datasets
>>>datasets()
```

**Comment by henrykironde** [comment]
Yes imports should be simpler.
I think the final usage may be something like, 

```python
import retriever as rp
rp.dataset()
 
```

**Comment by ShivamNegi** [comment]
I am thinking of adding all functions under the `datasets` module first and then working on making the import simpler. Cause I believe changing the way imports are being done will be another separate issue to work on.

**Comment by henrykironde** [comment]
Ok

File: retriever/lib/datasets.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from retriever import SCRIPT_LIST
+
+
+def datasets(arg_script=None):
+    """Function retruns list of all available datasets."""
```
Comment (henrykironde) [inline retriever/lib/datasets.py]: returns

Comment (henrykironde) [inline retriever/lib/datasets.py]: why do we need this parameter `arg_script=None`

Comment (ShivamNegi) [inline retriever/lib/datasets.py]: Changes made.

**Pull request merged**